holidays/countries/burkina_faso.py

```diff
@@ -24,6 +24,8 @@ class BurkinaFaso(ObservedHolidayBase, ChristianHolidays, InternationalHolidays,
     """
 
     country = "BF"
+    estimated_label = "%s (estimated)"
+    observed_estimated_label = "%s (observed, estimated)"
     observed_label = "%s (observed)"
```

Comment:
IMO translated `en_US` comment should be added whether or not that entity currently has l10n support or not, at least adding those later on would be simpler when needed
```suggestion
    # %s (estimated).
    estimated_label = "%s (estimated)"
    # %s (observed, estimated).
    observed_estimated_label = "%s (observed, estimated)"
    # %s (observed).
    observed_label = "%s (observed)"
```